src/kernels/jupyter/types.ts

```diff
@@ -235,9 +235,9 @@ export interface IJupyterServerUriStorage {
     getSavedUriList(): Promise<{ uri: string; time: number; displayName?: string }[]>;
     removeUri(uri: string): Promise<void>;
     clearUriList(): Promise<void>;
-    getUri(): Promise<string>;
     getRemoteUri(): Promise<string | undefined>;
```

Comment:
The plan is to use `getRemoteUri` instead of `getUri`, else we compare whehther its local or not and compare the string value.
This also paves the way to just have a simple function `getRemoteUri(): Promise<string[]>` which will return multiple uris if we have multiple remotes, and if its empty, then the conclusion is simple we don't have any remotes, (without that we need to compare strings, which is more of an internal implementation detail)